source/core_mqtt_serializer.c

```diff
@@ -962,12 +962,14 @@ static MQTTStatus_t deserializeConnack( const MQTTPacketInfo_t * pConnack,
              * "Session Present" bit is set. */
             if( pRemainingData[ 1 ] != 0U )
             {
+                LogError( ( "Session Present bit is set, but fourth byte in CONNACK is %u (nonzero).",
+                            ( unsigned int ) pRemainingData[ 1 ] ) );
                 status = MQTTBadResponse;
             }
         }
         else
         {
-            LogInfo( ( "CONNACK session present bit not set." ) );
+            LogDebug( ( "CONNACK session present bit not set." ) );
```

Comment:
This should be the same log level as the one for when the bit is set on line 958

Reply:
Changed LogInfo on line 958 to LogDebug to mirror line 970